CHANGELOG.md

```diff
@@ -20,7 +21,6 @@
 * [ENHANCEMENT] Added FIFO cache metrics for current number of entries and memory usage. #2270
 * [ENHANCEMENT] Output all config fields to /config API, including those with empty value. #2209
 * [BUGFIX] Fixed etcd client keepalive settings. #2278
-* [BUGFIX] Fixed bug in updating last element of FIFO cache. #2270
```

Comment:
Why this entry has been removed? Looks it was fixed in #2270, no?

Reply:
Since I changed LRU implementation, this BUGFIX does not apply anymore.